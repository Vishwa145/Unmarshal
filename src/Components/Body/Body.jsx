import React from 'react';
import Summary from "./Summary/Summary";
import TransactionTable from "./TransactionTable/TransactionTable";


function Body (){
    return(
        <div className="Body">
            <Summary />
            <TransactionTable />
        </div>
    );
}

export default Body;
