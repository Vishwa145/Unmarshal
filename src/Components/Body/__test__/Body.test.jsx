import React from "react";
import ReactDOM from "react-dom";
import Body from "../Body";
import renderer from "react-test-renderer"

it("render successfully", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Body />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("Matches snapshot", ()=>{
    const tree = renderer.create(<Body />).toJSON();
    expect(tree).toMatchSnapshot();
});