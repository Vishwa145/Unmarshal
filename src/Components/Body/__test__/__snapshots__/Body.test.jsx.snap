// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="Body"
>
  <div
    className="Summary"
  >
    <h3>
      <strong>
        Transaction Details
      </strong>
    </h3>
    <p>
      Unmarshal - sponsered slots available 
      <a
        href="/"
      >
        Book your slot here!
      </a>
    </p>
  </div>
  <div
    className="Container"
  >
    <div
      className="SwitchTab"
    >
      <button>
        <h3>
          Ethereum
        </h3>
      </button>
      <button>
        <h3>
          BSC
        </h3>
      </button>
      <button>
        <h3>
          Ethereum
        </h3>
      </button>
    </div>
    <h1>
      Loading
      ...
    </h1>
  </div>
</div>
`;
